### PR TITLE
Fixed the typo, BPG to BGP with all Docs

### PR DESCRIPTION
**Pull request by samkookji77**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by ctauchen** [comment]
Thanks @samkookji77. I notice that your find and replace removed a space character: 
image
You can update this PR with `git commit --amend` and force-push to your fork. No need to open a new PR.

File: calico-enterprise/getting-started/install-on-clusters/docker-enterprise.mdx [hunk 1]
```diff
@@ -134,7 +134,7 @@ watch kubectl get tigerastatus
 
 **Recommended - Networking**
 
-- The default networking uses IP-in-IP with BPG routing. For all networking options, see [Determine best networking option](../../networking/determine-best-networking.mdx).
+- The default networking uses IP-in-IP with BGProuting. For all networking options, see [Determine best networking option](../../networking/determine-best-networking.mdx).
```
Comment (ctauchen) [inline calico-enterprise/getting-started/install-on-clusters/docker-enterprise.mdx]: @samkookji77 I see that you force-pushed some changes after my last comment. But as you can see, the change didn't get into the revision. Can you try again?

Comment (samkookji77) [reply calico-enterprise/getting-started/install-on-clusters/docker-enterprise.mdx]: Hi, @ctauchen Do you want to change from 'BGProuting' to 'BGP routing'?

Comment (ctauchen) [reply calico-enterprise/getting-started/install-on-clusters/docker-enterprise.mdx]: Exactly. There should be a space: `BGP routing`, not `BGProuting`.

Comment (samkookji77) [reply calico-enterprise/getting-started/install-on-clusters/docker-enterprise.mdx]: @ctauchen Thank you for your reply. I fixed that typo and push it.

File: calico_versioned_docs/version-3.26/about/about-calico.mdx [hunk 2]
```diff
@@ -42,7 +42,7 @@ The following table summarizes the main {{prodname}} features. To search for spe
 | Feature                      | Description                                                  |
 | ---------------------------- | ------------------------------------------------------------ |
 | Dataplanes                   | eBPF, standard Linux iptables, Windows HNS, VPP.         |
-| Networking                   | &bull; Scalable pod networking using BPG or overlay networking<br />&bull; Advanced IP address management that is customizable |
+| Networking                   | &bull; Scalable pod networking using BGPor overlay networking<br />&bull; Advanced IP address management that is customizable |
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.26/about/about-calico.mdx]:
```suggestion
| Networking                   | &bull; Scalable pod networking using BGP or overlay networking<br />&bull; Advanced IP address management that is customizable |
```

File: calico/about/about-calico.mdx [hunk 3]
```diff
@@ -42,7 +42,7 @@ The following table summarizes the main {{prodname}} features. To search for spe
 | Feature                      | Description                                                  |
 | ---------------------------- | ------------------------------------------------------------ |
 | Dataplanes                   | eBPF, standard Linux iptables, Windows HNS, VPP.         |
-| Networking                   | &bull; Scalable pod networking using BPG or overlay networking<br />&bull; Advanced IP address management that is customizable |
+| Networking                   | &bull; Scalable pod networking using BGPor overlay networking<br />&bull; Advanced IP address management that is customizable |
```
Comment (ctauchen) [inline calico/about/about-calico.mdx]:
```suggestion
| Networking                   | &bull; Scalable pod networking using BGP or overlay networking<br />&bull; Advanced IP address management that is customizable |
```

**Review by ctauchen — COMMENTED** [review]
Very nearly there! Same issue, different word: `BGPor` ----> `BGP or`

File: calico/about/about-calico.mdx (hunk 3 above)
Comment (samkookji77) [reply calico/about/about-calico.mdx]: @ctauchen 
I fixed that typo and also fixed "iBGProuter" to "iBGP router" and push it.

File: calico-enterprise/reference/architecture/design/l3-interconnect-fabric.mdx [hunk 4]
```diff
@@ -232,7 +232,7 @@ As mentioned above, BGP expects that all of the iBGP routers in a network can se
...
+Conveniently, large scale/Internet scale networks solved this problem almost 20 years ago by deploying BGP route reflection as described in [RFC 1966](http://www.faqs.org/rfcs/rfc1966.html). This is a technique supported by almost all BGP routers today. In a large network, a number of route reflectors ([note 9](#note-9)) are evenly distributed and each iBGP router is _peered_ with one or more route reflectors (usually 2 or 3). Each route reflector can handle 10's or 100's of route reflector clients (in {{prodname}}'s case, the compute server), depending on the route reflector being used. Those route reflectors are, in turn, peered with each other. This means that there are an order of magnitude less route reflectors that need to be completely meshed, and each route reflector client is only configured to peer to 2 or 3 route reflectors. This is much easier to manage.
```
Comment (ctauchen) [inline calico-enterprise/reference/architecture/design/l3-interconnect-fabric.mdx]: Your find and replace introduced an error: `iBGProuter` must remain as it is. No space. Once this is fixed, we should be good to merge.

Comment (samkookji77) [reply calico-enterprise/reference/architecture/design/l3-interconnect-fabric.mdx]: @ctauchen 
oh, I am sorry. i fixed that and push it.
Thank you.

File: calico-cloud/reference/architecture/design/l3-interconnect-fabric.mdx [hunk 5]
```diff
@@ -435,7 +435,7 @@ routing protocols, or the like), and that routers in different ASs would
 not know how to reach one another unless they were directly connected.
 
 Based on that design point, routers in an iBGP peering relationship
-assume that they do not transit traffic for other iBGP routers in a
+assume that they do not transit traffic for other iBGProuters in a
```
Comment (ctauchen) [inline calico-cloud/reference/architecture/design/l3-interconnect-fabric.mdx:438]: @samkookji77 I'm afraid that your fix for the latest problem introduced a new error. There are places in the docs that need to be "iBGProuters" and others that need to be "iBGP routers". Your fix changed all instances to "iBGProuters".

I suggest you start over. In your branch, do a hard reset to the commit before your HEAD commit. Then do your global replacement (BPG --> BGP). Commit again, force push to origin. 

Lots of moving parts with even the smallest changes! Thanks for sticking with it.

**Comment by ctauchen** [comment]
Closing and reproducing in #768.

@samkookji77 Thanks again for pointing this out!

**Pull request closed without merging**